tests/phpunit/Utilities/Plugin_Request_Utility_Tests.php

```diff
@@ -171,4 +171,45 @@ public function test_destroy_runner_with_no_runner() {
 
 		$this->assertNull( $runner );
 	}
+
+	public function test_default_ignore_directories() {
```

Comment:
These tests are both nice additions. It would be great if we could add an additional integration test that can run against an example plugin in the [testdata](https://github.com/10up/plugin-check/tree/trunk/tests/phpunit/testdata/plugins) directory to ensure that ignored files are actually being ignored.

Reply:
Thanks @joemcgill I have added unit test as requested.